zschema/compounds.py

```diff
@@ -150,7 +150,7 @@ class SubRecord(Keyable):
 
     def __init__(self, definition=_NO_ARG, extends=_NO_ARG,
             allow_unknown=_NO_ARG, type_name=_NO_ARG, *args, **kwargs):
-        super(SubRecord, self).__init__(*args, **kwargs)
```

Comment:
???

Reply:
The superclass of `SubRecord` doesn't actually take any non-positional args, so if any `arg` comes through, then it crashes.

Reply:
Nevermind.